plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/steps/BuildLambda.kt

```diff
@@ -24,12 +27,14 @@ data class BuildLambdaRequest(
 
 class BuildLambda(private val request: BuildLambdaRequest) : SamCliStep() {
     override val stepName: String = message("lambda.create.step.build")
+    private val buildEnvVars = mapOf("GRADLE_USER_HOME" to gradleUserHomeDir)
```

Comment:
i dont think we always want to set the Gradle home directory to a temp directory. this should only be set if configured by a test